v5/owner.apib

```diff
@@ -0,0 +1,449 @@
+
++ Response 404
+Returned if parking space or period was not found, if the parking space doesn't belong to the user or period doesn't belong to the parking space.
+    + Body
+
+### Create new Times [POST /owner/parking-spaces/{parking_space_id}/periods/{period_id}/times]
++ Request (application/json)
+    + Attributes
+        + `weekday`: `1` (number, required)
+        + `time_start`: `00:00` (number, required)
+        + `time_end`: `19:00` (number, required)
+
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 201 (application/json)
+    + Attributes (Times)
+
++ Response 404
+Returned if parking space or period was not found or if the parking space doesn't belong to the user.
+    + Body
+
++ Response 422
+Returned if weekday is not in range 1-7, weekday already exists, time is invalid, weekday or times are missing, time is overlapping or an error occurred.
+    + Attributes (Api Error)
+
+### Update Times [PUT /owner/parking-spaces/{parking_space_id}/periods/{period_id}/times/{times_id}]
+Updates the times by setting the values of the parameters passed.
+
++ Parameters
+    + parking_space_id : `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+    + period_id : `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+    + times_id : `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+
++ Request  (application/json)
+    + Attributes
+        + `time_start`: `2017-12-15` (string) - can be null
+        + `time_end`: `null` (string) - can be null
+
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200
+    + Attributes (Times)
+
++ Response 404
+Returned if parking space, period or time was not found or if the parking space doesn't belong to the user.
+    + Body
+
++ Response 422
+Returned if weekday is not in range 1-7, weekday already exists, time is invalid, weekday or times are missing, time is overlapping or an error occurred.
+    + Attributes (Api Error)
+
+### Delete a Times [DELETE /owner/parking-spaces/{parking_space_id}/periods/{period_id}/times/{times_id}]
+
++ Parameters
+    + parking_space_id: `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+    + period_id: `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the period id
+    + times_id : `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+
++ Request
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200
+    + Attributes (Times)
+
++ Response 404
+Returned if parking space, period or time was not found or if the parking space doesn't belong to the user.
+    + Body
+
++ Response 422
+Returned if an error occurred.
+    + Body
+
+# Data Structures
+
+## Parking space (object)
++ `id`: `005c4826-4e28-11e3-a675-d43d7eece53d` (string)
++ `commission`: `33.33` (number, in percent)
```

Comment:
`in percent` is unfortunately not a valid type, but you can put this into the description, e.g.
```
`commission`: `33.33` (number) - commission in percent
```

Reply:
Done